internal/notification/static/i18n/zh.yaml

```diff
@@ -26,13 +26,19 @@ VerifyPhone:
   Greeting: 你好 {{.DisplayName}},
   Text: 您的用户中添加了一个新的手机号码，请使用以下验证码进行验证 {{.Code}}
   ButtonText: 验证手机号码
+VerifyEmailOTP:
+  Title: ZITADEL - 验证一次性密码
+  PreHeader: 验证一次性密码
+  Subject: 验证一次性密码
+  Greeting: 你好，{{.FirstName}} {{.LastName}}，
+  Text: 请使用 '验证' 按钮，或复制一次性密码 {{.OTP}} 并将其粘贴到验证屏幕中，以在接下来的五分钟内在 ZITADEL 中进行验证。
+  ButtonText: 验证
 DomainClaimed:
   Title: ZITADEL - 域名所有权验证
   PreHeader: 更改电子邮件/用户名
   Subject: 域名所有权验证
   Greeting: 你好 {{.DisplayName}},
   Text: 域 {{.Domain}} 已被组织使用。您当前的用户 {{.Username}} 不属于此组织。因此，您必须在登录时更改您的电子邮件。我们为此登录创建了一个临时用户名 ({{.TempUsername}})。
-  ButtonText: 登录
```

Comment:
why is that removed? and where is the sms text?

Reply:
Can we render buttons in SMS?

Reply:
LOL, idk

Reply:
readded these translations